src/gd_avif.c

```diff
@@ -477,7 +480,7 @@ static avifBool _gdImageAvifCtx(gdImagePtr im, gdIOCtx *outfile, int quality, in
 
 	uint32_t val;
 	uint8_t *p;
-	int x, y;
+	uint32_t x, y;
```

Comment:
Note that tests/avif/bad_input.c has an unused variable `retval`.

Reply:
good catch. I'll deal with that in a separate PR.